sway/handlers.c

```diff
@@ -334,7 +334,6 @@ static bool handle_key(wlc_handle view, uint32_t time, const struct wlc_modifier
 		release_key(sym, key);
 	}
 
-	// TODO: reminder to check conflicts with mod+q+a versus mod+q
```

Comment:
yes it should,
bindings are sorted from longest to shortest, so longer one have higher priority.
so alt+q+a gets checked before alt+q.
it was this one 470b4dfbae146d83c0061b39534c16b5aad90f1c